examples/practice-typescript/src/components/SideBar/index.scss

```diff
@@ -0,0 +1,13 @@
+@import '../../styles/bases/variable.scss';
```

Comment:
Should this belong to general file? We cannot duplicate import it every component.

Reply:
Fixed